### ISSUE TITLE
Frontmatter Date Added and Date Modified not populated

### ISSUE THREAD

**Issue by mtalexan** [platers/obsidian-linter#162]
In version 1.2.9.  
With the settings enabled to add the Date Added and Date Modified fields to the frontmatter, the frontmatter is not updated for cases where no frontmatter previously existed, or where frontmatter is of the following form (no other cases where tested, it might not work for other cases either):
```
---
tags: #tag1 #tag2 #tag3
---
```

**Comment by platers** [comment]
cant reproduce this either.

**Comment by mtalexan** [comment]
Hmm.  I could reproduce this one pretty easily before.  I had all settings enabled with defaults except these were turned off:
- Lint on Save
- Headings should be formatted with capitalization
- Ensures that footnote references are placed after punctuation, not before
- Replaces three consecutive dots with an elipse
- All paragraphs should have exactly one blank line both before and after 
- Removes leading and trailing blank lines in YAML front matter
- There should be at most one consecutive blank line
- Ensures that there is exactly one line break at the end of a document

I started trying to debug what was happening, so I created a new document "Test" at the top level and populated it with just:
```
---
tags: #tag1 #tag2 #tag3
---

```
I then disabled all my Community Plugins except the Linter, then ran "Linter: Lint the current file" from the command pallet and it worked as expected.  
So I started slowly adding Community Plugins back in and testing after each.  Even though it didn't work when I started, it started working once I'd gotten all plugins re-enabled.

It seems like there might be some weird interaction I was experiencing between plugins based on the order of installation/initialization.  Since it seems to be working now and I can't reproduce it any longer, I'll close this.  Sorry for the misleading bug.
